var/spack/repos/builtin/packages/py-tables/package.py

```diff
@@ -25,6 +26,7 @@ class PyTables(PythonPackage):
     variant("lzo", default=False, description="Support for lzo compression")
 
     # requirements.txt
+    depends_on("python@3.6:", when="@3.7.0:", type=("build", "run"))
     depends_on("python@3.5:", when="@3.6.1:", type=("build", "run"))
     depends_on("python@3.4:", when="@3.6.0:", type=("build", "run"))
     depends_on("python@2.6:", type=("build", "run"))
```

Comment:
Need deps from `pyproject.toml`:
```toml
requires = [
    "setuptools >=42.0",
    "wheel",
    "oldest-supported-numpy",
    "packaging",
    "Cython >=0.29.21",
]
```

Reply:
I will add py-oldest-supported-numpy later.

Reply:
@adamjstewart huh? https://pypi.org/project/oldest-supported-numpy/

Reply:
I  guess Spack handles that itself, right?